docs/source/commandline/commandline-all.rst

```diff
@@ -1190,108 +1612,138 @@ Core Vector Options
 
 **-beam_width_min** *<width>*
 
-	Sets the vector beam minimum width.
+    Sets the vector beam minimum width.
 
 .. _mame-commandline-beamwidthmax:
 
 **-beam_width_max** *<width>*
 
-	Sets the vector beam maximum width.
+    Sets the vector beam maximum width.
 
 .. _mame-commandline-beamintensityweight:
 
 **-beam_intensity_weight** *<weight>*
 
-	Sets the vector beam intensity weight.
+    Sets the vector beam intensity weight.
 
 .. _mame-commandline-flicker:
 
 **-flicker** *<value>*
 
-	Simulates a vector "flicker" effect, similar to a vector monitor that needs adjustment. This option requires a float argument in the range of 0.00 - 100.00 (0=none, 100=maximum). The default is ``0``.
+    Simulates a vector "flicker" effect, similar to a vector monitor that needs
+    adjustment.  This option requires a float argument in the range of
+    0.00 - 100.00 (0=none, 100=maximum).
 
+    The default is ``0``.
 
 
 Core Video OpenGL Debugging Options
 -----------------------------------
 
-These options are for compatibility in **-video opengl**. If you report rendering artifacts you may be asked to try messing with them by the devs, but normally they should be left at their defaults which results in the best possible video performance.
+These options are for compatibility in **-video opengl**.  If you report
+rendering artifacts you may be asked to try messing with them by the devs, but
+normally they should be left at their defaults which results in the best
+possible video performance.
 
 .. _mame-commandline-glforcepow2texture:
 
 **-[no]gl_forcepow2texture**
 
-	Always use only power-of-2 sized textures (default ``off``)
+    Always use only power-of-2 sized textures.
+
+    The default is OFF. (**-nogl_forcepow2texture**)
 
 .. _mame-commandline-glnotexturerect:
 
 **-[no]gl_notexturerect**
 
-	Don't use OpenGL GL_ARB_texture_rectangle (default ``on``)
+    Don't use OpenGL GL_ARB_texture_rectangle.
+
+    The default is ON. (**-gl_notexturerect**)
 
 .. _mame-commandline-glvbo:
 
 **-[no]gl_vbo**
 
-    Enable OpenGL VBO (Vertex Buffer Objects), if available (default ``on``)
+    Enable OpenGL VBO (Vertex Buffer Objects), if available.
+
+    The default is ON. (**-gl_vbo**)
 
 .. _mame-commandline-glpbo:
 
 **-[no]gl_pbo**
 
     Enable OpenGL PBO (Pixel Buffer Objects), if available (default ``on``)
 
+    The default is ON. (**-gl_pbo**)
+
 
 Core Video OpenGL GLSL Options
 ------------------------------
 
-
 .. _mame-commandline-glglsl:
 
 **-gl_glsl**
 
-	Enable OpenGL GLSL, if available (default ``off``)
+    Enable OpenGL GLSL, if available.
+
+    The default is OFF.
 
 .. _mame-commandline-glglslfilter:
 
 **-gl_glsl_filter**
 
-	Enable OpenGL GLSL filtering instead of FF filtering -- *0-plain, 1-bilinear, 2-bicubic* (default is ``1``)
+    Enable OpenGL GLSL filtering instead of FF filtering.
```

Comment:
probably better to expand the abbreviation - something like "Use an OpenGL GLSL shader to perform filtering, rather than using the fixed function pipeline's bilinear filtering."

Reply:
Done.